documentation/modules/upgrading/proc-downgrade-cluster-operator.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// assembly-downgrade.adoc
+
+[id='proc-downgrade-cluster-operator-{context}']
+= Downgrading the Cluster Operator to a previous version
+
+If you are encountering issues with the latest version of Strimzi,
```

Comment:
Same as earlier ... the _latest version_ sounds weird and confusing here.

Reply:
Changed to:

```
If you are encountering issues with Strimzi,
you can revert your installation.
```